Altis_Life.Altis/core/functions/TON/fn_isNumber.sqf

```diff
@@ -0,0 +1,15 @@
+/*
+    File: fn_isNumber.sqf
+    Author: Bryan "Tonic" Boardwine
+
+    Description: determines whether or not a string represents a valid number
+*/
+params [
+    ["_string","",[""]]
+];
+if (_string isEqualTo "") exitWith {false};
+private _array = _string splitString "";
+
+{
+    !(_x in ["0","1","2","3","4","5","6","7","8","9"]);
+} count _array > 0;
```

Comment:
findIf instead?

Reply:
thinking about it... why not just use [BIS_fnc_parseNumber](https://community.bistudio.com/wiki/BIS_fnc_parseNumber) instead? i  mean, it already exists and returns `-1` if its not a number

Reply:
I remember there was a reason we didn't use the BIS function - although it escapes me right now...

Reply:
okay, so no BIS_fnc_parseNumber.

But thinking about it, can you even do it with findIf? I mean, we would have to check if if has anything **but** a number in it. Sadly, Arma doesn't recognize RegEx, otherwise this would be pretty easy :D `\d*`

Reply:
(_array findIf {_x in ["0","1","2","3","4","5","6","7","8","9"]}) != -1

Reply:
`(_array findIf {!(_x in ["0","1","2","3","4","5","6","7","8","9"])}) isEqualTo -1` would be correct (tested in editor. your version if true as soon as there is a number anywhere in it)

testing with 10,000 executions showed no benefit, so I think this change is not really that necessary?

Reply:
Really doesn't matter which it is